### PR TITLE
Add ossfs

### PR DESCRIPTION
**Pull request by karajan1001**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by efiop** [comment]
>  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

I confirm :slightly_smiling_face:

File: recipes/ossfs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - oss2 ==2.6.1
```
Comment (efiop) [inline recipes/ossfs/meta.yaml:22]: @karajan1001  Should we relax this to `>=` here and in original project?

**Comment by efiop** [comment]
@conda-forge/staged-recipes Hey guys, just a very gentle ping that this is now ready to be reviewed when you have time. :slightly_smiling_face:

File: recipes/ossfs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ossfs-{{ version }}.tar.gz
+  sha256: 7d5b9396baa681748c2d599d6006da29ecf140c99da3f5daec2b366d77737778
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/ossfs/meta.yaml:15]: Can this package be `noarch`?

Comment (efiop) [reply recipes/ossfs/meta.yaml:15]: Good catch!
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```

File: recipes/ossfs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (efiop) [inline recipes/ossfs/meta.yaml:20]:
```suggestion
    - python >=3.6
```

File: recipes/ossfs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - oss2 >=2.6.1
+    - fsspec >=2021.7.0
+    - python
```
Comment (efiop) [inline recipes/ossfs/meta.yaml:24]:
```suggestion
    - python >=3.6
```

**Comment by efiop** [comment]
@ocefpaf Thanks again for the help! Ready for another round of reviews :slightly_smiling_face:

**Comment by efiop** [comment]
@conda-forge/staged-recipes Hey guys, just a very gentle ping that this is now ready to be reviewed when you have time. :slightly_smiling_face:

**Review by dopplershift — APPROVED** [review]

**Pull request merged**